.github/workflows/ci.yml

```diff
@@ -812,49 +815,69 @@ jobs:
         with:
-          python-version: 3.9
+          library-name: ${{ env.PACKAGE_NAME }}
+          python-version: ${{ env.MAIN_PYTHON_VERSION }}
+
+
+  # release-bis:
+  #   name: "Release"
+  #   if: github.event_name == 'push' && contains(github.ref, 'refs/tags')
+  #   needs: [docs-build, build-test, build-test-ubuntu, build-test-ubuntu-minimal]
+  #   runs-on: ubuntu-latest
+  #   steps:
+  #     - name: Set up Python
+  #       uses: actions/setup-python@v5
+  #       with:
+  #         python-version: 3.9
 
-      - uses: actions/download-artifact@v4
+  #     - uses: actions/download-artifact@v4
       
-      - name: Copying files and overwrite same name files
-        # (to avoid upload issue)
-        run: |
-          mkdir artifacts
-          find . -iname 'ansys_mapdl_core-*.tar.gz' -exec cp \{\} ./artifacts/ \;
-          find . -iname 'ansys_mapdl_core-*-py3-none-any.whl' -exec cp \{\} ./artifacts/ \;
-          find . -iname 'minimum_requirements.txt' -exec cp \{\} ./artifacts/ \;
+  #     - name: Copying files and overwrite same name files
+  #       # (to avoid upload issue)
+  #       run: |
+  #         mkdir artifacts
+
+  #     - name: Display structure of downloaded files
+  #       run: ls -Rla
 
-      - name: Display structure of downloaded files
-        run: ls -Rla
+  #     - name: "Release to GitHub"
+  #       uses: softprops/action-gh-release@v2
+  #       with:
+  #         files: |
+  #           ./**/pymapdl-Documentation-*.pdf
+  #           ./**/ansys-mapdl-core*wheelhouse*.zip
 
-      - name: "Release to GitHub"
-        uses: softprops/action-gh-release@v2
+
+  release:
+    name: Release project
+    if: ${{ github.event_name == 'push' && contains(github.ref, 'refs/tags') }}
+    needs: [package]
+    runs-on: ubuntu-latest
+    # Specifying a GitHub environment is optional, but strongly encouraged
+    environment: release
+    permissions:
+      id-token: write
+      contents: write
+    steps:
+      - name: Release to the public PyPI repository
+        uses: ansys/actions/release-pypi-public@v6
         with:
-          files: |
-            ./**/artifacts/ansys_mapdl_core-*-py3-none-any.whl
-            ./**/artifacts/ansys_mapdl_core-*.tar.gz
-            ./**/artifacts/minimum_requirements.txt
-            ./**/pymapdl-Documentation-*.pdf
-            ./**/ansys-mapdl-core*wheelhouse*.zip
+          library-name: ${{ env.PACKAGE_NAME }}
+          use-trusted-publisher: true
 
-      - name: Upload to Public PyPi
-        env:
-          TWINE_USERNAME: __token__
-          TWINE_PASSWORD: ${{ secrets.PYPI_TOKEN }}
-        run: |
-          pip install twine
-          twine upload --skip-existing ./**/*.whl
-          twine upload --skip-existing ./**/*.tar.gz
+      - name: Release to GitHub
+        uses: ansys/actions/release-github@v6
+        with:
+          library-name: ${{ env.PACKAGE_NAME }}
+          additional-artifacts: "minimum_requirements.txt"

```

Comment:
I assume the `ansys` actions take care of the artifacts properly. I did check for `documentation-pdf`, `documentation-html` and the wheelhouse files.
I guess that's all.